simple/template_schema.py

```diff
@@ -0,0 +1,159 @@
+
+# instead of having modes...telescope --> instrument --> mode (used mostly for spectra). think about SVO.
+class Modes(Base):
+    __tablename__ = 'Modes'
+    mode = Column(String(30), primary_key=True, nullable=False)
+    instrument = Column(String(30), ForeignKey('Instruments.instrument', onupdate='cascade'), primary_key=True)
+    telescope = Column(String(30), ForeignKey('Telescopes.telescope', onupdate='cascade'), primary_key=True)
+    description = Column(String(1000))
+
+
+class PhotometryFilters(Base):
+    """
+    ORM for filter table.
+    This stores relationships between filters and instruments, telescopes, as well as wavelength and width
+    """
+    __tablename__ = 'PhotometryFilters'
+    band = Column(String(30), primary_key=True, nullable=False)  # of the form instrument.filter (see SVO)
+    instrument = Column(String(30), ForeignKey('Instruments.name', onupdate='cascade'), primary_key=True)
+    telescope = Column(String(30), ForeignKey('Telescopes.name', onupdate='cascade'), primary_key=True)
+    effective_wavelength = Column(Float, nullable=False)
+    width = Column(Float)
+
+
+class Versions(Base):
+    """
+    ORM for Versions table
+    This stores the version numbers for the database
+    """
+    __tablename__ = 'Versions'
+    version = Column(String(30), primary_key=True, nullable=False)
+    start_date = Column(String(30))
+    end_date = Column(String(30))
+    description = Column(String(1000))
+
+
+# -------------------------------------------------------------------------------------------------------------------
+# Hard-coded enumerations
+
+class Regime(enum.Enum):
+    """Enumeration for spectral type, spectra, and photometry regimes
+    Use UCD controlled vocabulary: https://www.ivoa.net/documents/UCD1+/20200212/PEN-UCDlist-1.4-20200212.html#tth_sEcB
+    The string values are used, not the variable names.
+    """
+    ultraviolet = 'em.UV'
+    optical_UCD = 'em.opt'
+    optical = 'optical'
+    nir_UCD = 'em.IR.NIR'  # Near-Infrared, 1-5 microns
+    nir = 'nir'
+    infrared = 'em.IR'  # Infrared part of the spectrum
+    mir_UCD = 'em.IR.MIR'  # Medium-Infrared, 5-30 microns
+    mir = 'mir'
+    millimeter = 'em.mm'
+    radio = 'em.radio'
+    unknown = 'unknown'
+
+# -------------------------------------------------------------------------------------------------------------------
+# Main tables
+class Sources(Base):
+    """ORM for the sources table. This stores the main identifiers for our objects along with ra and dec"""
+    __tablename__ = 'Sources'
+    source = Column(String(100), primary_key=True, nullable=False)
+    ra = Column(Float)
+    dec = Column(Float)
+    epoch = Column(Float)  # decimal year
+    equinox = Column(String(10))  # eg, J2000
+    shortname = Column(String(30))  # not needed?
+    # reference = Column(String(30), ForeignKey('Publications.publication', onupdate='cascade'), nullable=False)
+    other_references = Column(String(100))
```

Comment:
Curious why you would remove reference in favor of other_reference. If all you wanted was to remove the ForeignKey relationship, it's easier to do just that than create a new column.